Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.Members.cs

```diff
@@ -0,0 +1,178 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Numerics;
+using Windows.ApplicationModel;
+using Windows.Foundation.Metadata;
+using Windows.UI;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Shapes;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="DropShadowPanel"/> control allows the creation of a DropShadow for any Xaml FrameworkElement in markup
+    /// making it easier to add shadows to Xaml without having to directly drop down to Windows.UI.Composition APIs.
+    /// </summary>
+    public partial class DropShadowPanel
+    {
+        /// <summary>
+        /// Gets a value indicating whether the platform supports drop shadows.
+        /// </summary>
+        /// <remarks>
+        /// On platforms not supporting drop shadows, this control has no effect.
+        /// </remarks>
+        public static bool IsSupported =>
+            !DesignMode.DesignModeEnabled &&
+            ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 3); // SDK >= 14393
+
+        /// <summary>
+        /// Gets or sets the casting element.
+        /// </summary>
+        public FrameworkElement CastingElement
```

Comment:
I don't get what a casting element is - I'm guessing might not either. Could you add a little more to the summary?